picard/tagger.py

```diff
@@ -973,8 +972,13 @@ def main(localedir=None, autoupdate=True):
         return longversion()
 
     if not (IS_WIN or IS_MACOS or IS_HAIKU):
-        dbus = QDBusConnection.sessionBus()
-        dbus.registerService(PICARD_APP_ID)
+        try:
+            from PyQt5.QtDBus import QDBusConnection
+        except ImportError:
+            pass
+        else:
+            dbus = QDBusConnection.sessionBus()
+            dbus.registerService(PICARD_APP_ID)
```

Comment:
What about getting rid of platform checks?

```python
        try:
            from PyQt5.QtDBus import QDBusConnection
            dbus = QDBusConnection.sessionBus()
            dbus.registerService(PICARD_APP_ID)
        except ImportError:
             dbus = None
```       

And later check with `if dbus:`

@phw : any drawback with this approach?

Reply:
(Was wondering about that, too.)

Reply:
I just wanted to avoid that we register the service on DBus on platforms where we definitely know we will not need it. It's likely very uncommon to find this, but dbus is potentially available for Windows. Similar for Haiku. No idea about macOS.

But yeah, we could probably remove the check here. I would however not remove the check from statusindicator.py. There is really no need to even try to setup UnityLauncherEntryStatusIndicator on those platforms.